website/docusaurus.config.js

```diff
@@ -13,7 +13,7 @@ export default async function createConfigAsync() {
   return {
     title: "RISC Zero Developer Docs",
     tagline: "Hyper-Efficient General Purpose Zero-Knowledge Computing.",
-    favicon: "img/logo.png",
+    favicon: "img/favicon.ico",
```

Comment:
![image](https://github.com/user-attachments/assets/db66f9cd-aa4f-403a-8865-2bdf86d5832b)

is this one still not right?